docs/_admin/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Project Administration
+order: -1
+---
+
+The web-platform-tests project is more than just tests! This section documents
+all the information necessary to administer the infrastructure which makes the
+project possible.
+
+## Secrets
+
+Some aspects of the infrastructure are only accessible to administrators. These
+include access credentials for third-party services (e.g. continuous
+integration providers), data used to prove identity (e.g. SSL certificates),
+and deployment information for the project's running systems (e.g.
+[w3c-test.org](https://w3c-test.org)).
+
+Project                   | Secret                   | Owners
+--------------------------|--------------------------|-------------
+[results-collection]      | root SSH keys            | boaz@bocoup.com, mike@bocoup.com, rick@bocoup.com
+[results-collection]      | Password for app secrets | boaz@bocoup.com, mike@bocoup.com, rick@bocoup.com
+[pull-requests-dashboard] | root SSH keys            | ???
+[pull-requests-dashboard] | Password for app secrets | boaz@bocoup.com, geoffers@gmail.com, jgraham@hoppipolla.co.uk, lukebjerring@google.com, mike@w3.org
+[web-platform-tests]      | TravisCI authorization   | ???
+
+Note that SSL certificates for all HTTPS-enabled domains are retrieved via
+[Let's Encrypt](https://letsencrypt.org/), so that data does not represent an
+explicitly-managed secret.
+
+## Third-party account owners
+
+- (unknown registrar): https://web-platform-tests.org
+  - ???
+- (unknown registrar): https://w3c-test.org
+  - ???
+- (unknown registrar): http://testthewebforward.org
+- [Google Domains](https://domains.google/): https://wpt.fyi
+  - foolip@google.com
+  - jeffcarp@google.com
+  - lukebjerring@google.com
+  - mike@bocoup.com
+- [GitHub](hittps://github.com): web-platform-tests
+  - [@foolip](https://github.com/foolip)
+  - [@Hexcles](https://github.com/Hexcles)
+  - [@jgraham](https://github.com/jgraham)
+  - [@plehegar](https://github.com/plehegar)
+  - [@thejohnjansen](https://github.com/thejohnjansen)
+  - [@youennf](https://github.com/youennf)
+  - [@zcorpan](https://github.com/zcorpan)
+- [GitHub](https://github.com): w3c
+  - ??? (soon to be obviated)
```

Comment:
should be web-human@w3.org as well. Folks are always welcome to reach out to me and/or @sideshowbarker in any case.

Reply:
Thanks, Philippe! I'm preferring references to real people when possible because this is the first step in a larger effort to manage all these accounts.